uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml

```diff
@@ -84,6 +85,30 @@ Page {
      */
     property var popup: null
 
+    /*!
+       \brief Boolean that controls if the PopupView will attempt to open URL's with an external browser.
+
+       Defaults to true.
+       \qmlproperty bool openUrlsExternally
+     */
+    property bool openUrlsExternally: true
+
+    /*!
+       \brief Boolean that controls if the PopupView will attempt to open images with an internal viewer.
+
+       Defaults to true.
+       \qmlproperty bool openImagesInternally
+     */
+    property bool openImagesInternally: true
```

Comment:
maybe `openImagesInApp` or similar would be a little more descriptive?

Reply:
I feel similarly about these. It is not obvious what externally and internally mean.

Reply:
@ldanzinger @JamesMBallard how about `openUrlsWithSystemDefaultApplication`. It's long but descriptive. Attachments would be `openAttachmentsWithSystemDefaultApplication`.

Reply:
>how about openUrlsWithSystemDefaultApplication. It's long but descriptive. Attachments would be openAttachmentsWithSystemDefaultApplication.

I like that better.

Reply:
Yeah, something along those lines works for me